### PR TITLE
5904 regenerate sdk 01

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
This pull request adds stopwords support to the Discovery service, grammars support to the Speech to Text service and language support to the VisualRecognition service's detect faces method.

### CODE REVIEW

File: Scripts/Services/Assistant/v1/Assistant.cs [hunk 1]
```diff
@@ -4444,7 +4444,7 @@ private void OnUpdateDialogNodeResponse(RESTConnector.Request req, RESTConnector
         /// </summary>
         /// <param name="successCallback">The function that is called when the operation is successful.</param>
         /// <param name="failCallback">The function that is called when the operation fails.</param>
-        /// <param name="filter">A cacheable parameter that limits the results to those matching the specified filter. You must specify a filter query that includes a value for `language`, as well as a value for `workspace_id` or `request.context.metadata.deployment`. For more information, see the [documentation](https://console.bluemix.net/docs/services/conversation/filter-reference.html#filter-query-syntax).</param>
+        /// <param name="filter">A cacheable parameter that limits the results to those matching the specified filter. You must specify a filter query that includes a value for `language`, as well as a value for `workspace_id` or `request.context.metadata.deployment`. For more information, see the [documentation](https://cloud.ibm.com/docs/services/conversation/filter-reference.html#filter-query-syntax).</param>
```
Comment (mamoonraja) [inline Scripts/Services/Assistant/v1/Assistant.cs]: conversation -> assistant

File: Scripts/Services/Assistant/v1/Models/CreateDialogNode.cs [hunk 2]
```diff
@@ -164,9 +164,9 @@ public enum EventNameEnum
         [fsProperty("previous_sibling")]
         public string PreviousSibling { get; set; }
         /// <summary>
-        /// The output of the dialog node. For more information about how to specify dialog node output, see the [documentation](https://console.bluemix.net/docs/services/conversation/dialog-overview.html#complex).
+        /// The output of the dialog node. For more information about how to specify dialog node output, see the [documentation](https://cloud.ibm.com/docs/services/conversation/dialog-overview.html#complex).
```
Comment (mamoonraja) [inline Scripts/Services/Assistant/v1/Models/CreateDialogNode.cs]: I think we were changing `docs/services/conversation` to assistant

File: Scripts/Services/Assistant/v1/Models/CreateValue.cs [hunk 3]
```diff
@@ -73,9 +73,9 @@ public enum ValueTypeEnum
         [fsProperty("synonyms")]
         public List<string> Synonyms { get; set; }
         /// <summary>
-        /// An array of patterns for the entity value. You can provide either synonyms or patterns (as indicated by **type**), but not both. A pattern is a regular expression no longer than 128 characters. For more information about how to specify a pattern, see the [documentation](https://console.bluemix.net/docs/services/conversation/entities.html#creating-entities).
+        /// An array of patterns for the entity value. You can provide either synonyms or patterns (as indicated by **type**), but not both. A pattern is a regular expression no longer than 128 characters. For more information about how to specify a pattern, see the [documentation](https://cloud.ibm.com/docs/services/conversation/entities.html#creating-entities).
```
Comment (mamoonraja) [inline Scripts/Services/Assistant/v1/Models/CreateValue.cs]: doc links as mentioned above

File: Scripts/Services/Assistant/v1/Assistant.cs (hunk 1 above)
Comment (mediumTaj) [reply Scripts/Services/Assistant/v1/Assistant.cs]: ahh youre right i didnt go far enough in the url. Fixing

File: Examples/ServiceExamples/Scripts/ExampleDiscovery.cs [hunk 4]
```diff
@@ -333,6 +346,46 @@ private IEnumerator Examples()
...
+                yield return null;
+        }
+
+        //  Delete stopword list
+        Log.Debug("ExampleDiscovery.RunTest()", "Attempting to delete stopword list {0}", _createdCollectionId);
+        _service.DeleteStopwordList(OnDeleteStopwordList, OnFail, _environmentId, _createdCollectionId);
+        while (!_deleteStopwordListTested)
+            yield return null;
+
+        //  List Gatways
```
Comment (mamoonraja) [inline Examples/ServiceExamples/Scripts/ExampleDiscovery.cs]: *Gateways

**Pull request merged**